projects/com_protocol/src/i2c_spi_test.c

```diff
@@ -0,0 +1,169 @@
+#include <stdbool.h>
+#include "gpio.h"
+#include "i2c.h"
+#include "log.h"
+#include "spi.h"
+
+// Part 1: I2C
+
+// Set this to true to perform an I2C write.
+#define SHOULD_WRITE true
+#define WRITE_I2C_PORT I2C_PORT_1  // or I2C_PORT_2
+#define WRITE_I2C_ADDRESS 0x48

```

Comment:
I suppose it's okay that you copied the I2C and SPI smoke tests for the homework, but do make sure you understand how to use the I2C and SPI libraries and how the I2C and SPI protocols work.